internal/applicationsnapshot/input.go

```diff
@@ -97,6 +112,8 @@ func DetermineInputSpec(ctx context.Context, input Input) (*app.SnapshotSpec, er
 		if err != nil {
 			return nil, err
 		}
+		log.Debugf("Calling expandImageIndex")
```

Comment:
Logging here without any detail and only for one branch of the execution doesn't add value, perhaps adding logging statements to `expandImageIndex` makes more sense